frontend/public/components/utils/kebab.tsx

```diff
@@ -550,13 +550,13 @@ class KebabWithTranslation extends React.Component<
     const menuOptions = kebabOptionsToMenu(options);
 
     return (
-      <div
-        className={classNames({
-          'pf-c-dropdown': true,
-          'pf-m-expanded': this.state.active,
-        })}
-      >
-        <Tooltip content={hoverMessage} isVisible={isDisabled && !!hoverMessage} trigger="manual">
+      <Tooltip content={hoverMessage} trigger="mouseenter">
```

Comment:
Why are we moving all the other components inside the Tooltip component? Shouldn't it only contain the `button` and `EllipsisVIcon`?

Reply:
Hello, @jhadvig. I actually tried that first, but that did not seem to work, so I had to go outward on the scope.